CollapseLauncher/Classes/InstallManagement/Genshin/GenshinInstall.cs

```diff
@@ -22,7 +22,8 @@ internal class GenshinInstall : InstallManagerBase<GameTypeGenshinVersion>, IGam
         protected override int _gameVoiceLanguageID { get => _gameVersionManager.GamePreset.GetVoiceLanguageID(); }
         #endregion
 
-        #region Private Properties
+        #region Properties
+        public bool StartAfterInstall { get; set; } = false;
```

Comment:
To make a future use of this property, I would suggest to put this ``StartAfterInstall`` property under "Public Properties" region inside of ``InstallManagerBase<T>`` base class.

Reply:
Reshaper was complaining for some reason when I tried that... Now it doesn't...
Changed it too.